config/application.rb

```diff
@@ -28,5 +28,8 @@ class ApplicationLetter < Rails::Application
 
     # see Event::unreasonably_long, unit in days
     config.unreasonably_long_event_time_span = 300
+
+    # default reply_to email adress
+    config.default_reply_to_email = 'workshop-portal@gmail.com'
```

Comment:
If the system is supposed to run at/for the HPI, they probably prefer an @hpi.de adress as default reply-to?
@tobiduer?